js_modules/dagster-ui/packages/ui-core/src/nav/__tests__/ScheduleOrSensorTag.test.tsx

```diff
@@ -0,0 +1,182 @@
+import {render, screen} from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import * as React from 'react';
+import {MemoryRouter} from 'react-router';
+
+import {
+  InstigationStatus,
+  buildInstigationState,
+  buildSchedule,
+  buildSensor,
+} from '../../graphql/types';
+import {DUNDER_REPO_NAME, buildRepoAddress} from '../../workspace/buildRepoAddress';
+import {ScheduleOrSensorTag} from '../ScheduleOrSensorTag';
+
+describe('ScheduleOrSensorTag', () => {
```

Comment:
This was being tested via `JobMetadata` with some complex mocking, but makes more sense as its own test, especially since the component accepts fragments and doesn't otherwise do any querying of its own.